### DISCUSSION TITLE
Help understanding strings for translation purpose

### DISCUSSION THREAD

**Discussion by ezellohar** [windingwind/zotero-better-notes#1132] [General]
Hello, I'm working to make the italian translation up-to-date, but I have some troubles figuring out the scope of some strings.
File: templateEditor.ftl
Starting from line 78, what are those strings for? JS script or are they related to BN and MD files sync?
Also, when I find something like: `.title = Type: string` could I translate it or should I leave it as it is (same goes for `Type: Zotero.Item`)?
Thanks

**Comment by windingwind** [comment]
Hi, thanks for your contribution. They’re for the new feature, template editor snippet. These are the tooltips for the snippet buttons, you can hover them in the template editor to see them.

I will update the ftl files to make them more friendly to contributors. Thanks for letting me know!

**Reply by windingwind** [reply]
I updated the ftl files. Take the en-US as an example:

https://github.com/windingwind/zotero-better-notes/blob/fca3899e8d13608b7150cebec5c87a6df155eb10/addon/locale/en-US/templateEditor.ftl#L87-L96

Here the `{ -variable-type }` is from the string defined above. Skip content inside the curly braces and translate the rest of the strings.
